### ISSUE TITLE
Launcher font is just rectangles in Linux Mint Xfce 21.2

### ISSUE THREAD

**Issue by KreaeTerra** [qrrk/Catapult#131]
I ran the latest 23.03a version of the launcher on my laptop running linux mint xfce 21.2 but the interface font of the launcher is just rectangles and nothing else.
I'm not that well versed with linux yet though so it might just be something im doing wrong on my end. Not too sure.
![screenshot](https://github.com/qrrk/Catapult/assets/92649837/7782de4c-15b5-4483-9846-d1cde459d955)

**Comment by qrrk** [comment]
Looks like broken font scaling in the engine...

**Comment by KreaeTerra** [comment]
Yeah I assumed something like that. It works fine on my main pc running the same os but the one with the isse is is my very old laptop with a low resolution monitor so idk if that could cause something like this.

 Also I'm real inexperienced with GitHub though so sorry if i gave too little information or something.

**Comment by somichev-dev** [comment]
Could not replicate that with the latest version of the launcher. Perhaps, this was fixed by Godot 4 migration?
